### ISSUE TITLE
Add/Remove cell within a table

### ISSUE THREAD

**Issue by strandbygaard** [tgrosinger/advanced-tables-obsidian#230]
**As-is** It is possible to add or remove rows and columns. It is not possible to add or remove cells within the table. Adding a single cell within a table can be a quite big manual copy-paste task.

**To-be** it would be awesome with a feature similar to Excel, where you can add (or remove) a single cell within an existing table, and decide if the existing cells should be shifted down or right. Shifting down would imply adding an additional row. Shifting right would imply adding an additional column. 

Similar to this part of the Excel "insert" interface.
<img width="272" alt="image" src="https://user-images.githubusercontent.com/27151/209426951-39089658-bc38-457e-a991-5c3dd8c7c072.png">

**Comment by elzibubble** [comment]
Yes please! This would be so helpful. I sometimes make character sheets in Obsidian with metabind, and this means I not infrequently use tables with (what would be ideally represented as) multiple column-sets.

Ideally this would also include the ability to *delete* a single cell, while shifting values within the row/col to the left/up so that no blank is left.
